### DISCUSSION TITLE
Add `bodyweight` variable

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#122] [Ideas]
So that you could use your current body weight in the progression scripts.

**Comment by yrinkel** [comment]
Great idea. A bodyweight variable that can be used in scripts. Bonus points if it’s automatically read from the “measures”. Extra bonus points if you get the (moving) average weight as configured in “Measures” 😊

**Comment by grooves12** [comment]
What I'd like to see is to be able to designate exercises as bodyweight exercises, which then automatically add bodyweight to calculate volume. This should allow for negative numbers in the weights for assisted exercises such as pullups/dips.

**Comment by astashov** [comment]
Ok done!
https://www.reddit.com/r/liftosaur/comments/1lhree7/added_bodyweight_variable/

**Comment by yrinkel** [comment]
And it works! And uses moving average! You rock!!
